### PR TITLE
Rewrote script in Python

### PR DESCRIPTION
**Pull request by tuxudo**
Also reorganized UI views

### CODE REVIEW

**Comment by bochoven** [comment]
I'm afraid I messed up this PR with my latest code changes. We should check if my edits did not break the data from the python script.

**Comment by tuxudo** [comment]
Finally have the conflicts resolved and tested, PR ready to be merged

File: power_controller.php [hunk 1]
```diff
@@ -48,15 +48,15 @@ public function get_data($serial_number = '')
      **/
     public function get_stats()
     {
-        $out = array();
+        $obj = new View();
+        
         if (! $this->authorized()) {
```
Comment (bochoven) [inline power_controller.php]: You can get rid of all the $this-authorized() calls. All methods are now protected except get_script() and index()

**Review by bochoven — COMMENTED** [review]
Please rework all parts where Mac data is rendered inside .html()

**Comment by tuxudo** [comment]
Removed html calls

**Pull request merged**